docs/relay/relay.md

```diff
@@ -64,96 +83,70 @@ if (!AuthenticationService.Instance.IsSignedIn)
 }
 ```
 
-Here's the complete code that allows working with the [Relay server](https://docs.unity.com/relay/en/manual/relay-servers).
+## How do I join an existing allocation?
```

Comment:
```suggestion
## Join an existing allocation
```

Reply:
Just so I understand what's happening here - Is this the process of joining a game that another player is hosting on their IP address?

Reply:
In a way :-)
It means that you connect to the relay server to join the game, and the relay server is going to remember your IP. The IP of the host is not disclosed to the client connecting.
The typical use-case is indeed one player being the host and acting as the "game server", but it is also possible to use an actual server instead of a player.